airbyte-webapp/src/pages/routes.tsx

```diff
@@ -152,6 +163,16 @@ export const Routing = () => {
         ) : (
           <MainViewRoutes />
         )}
+        <ChatWidget
+          title="Welcome to Airbyte"
+          subtitle="Ask us anything in the chat window below 😊"
+          primaryColor="#625eff"
+          greeting="Hello!!!"
+          newMessagePlaceholder="Start typing..."
+          customer={customer}
+          accountId="74560291-451e-4ceb-a802-56706ece528b"
+          baseUrl="https://app.papercups.io"
+        />
```

Comment:
(if you're curious, we also have a bunch of additional props that might be useful https://github.com/papercups-io/chat-widget#options 😄 and feel free to let me know if there's anything else you'd like control/customization over!)